lib/pgrel/active_record/store_chain.rb

```diff
@@ -120,6 +128,16 @@ def invert_arel(rel)
           end
         end
       end
+
+      private
+
+      def contains_clause(opts)
+        "#{@store_name} @> #{type_cast(opts)}"
+      end
+
+      def build_or_contains(k, vals)
+        vals.map { |v| contains_clause(k => v) }.join(' or ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.